src/l10n/fr.js

```diff
@@ -0,0 +1,357 @@
+            'Username'                            : 'Nom d’utilisateur',
+            'Website'                             : 'Site internet',
+            'Add Tags...'                         : 'Ajouter des mots clés…',
+            'Color'                               : 'Couleur',
+            'Name'                                : 'Nom',
+            'Label'                               : 'Étiquette',
+            'General'                             : 'Général',
+            'Properties'                          : 'Propriétés',
+            'Numbers'                             : 'Nombres',
+            'Special Characters'                  : 'Caractères spéciaux',
+            'More Options'                        : 'Plus d’options',
+            'Favorite'                            : 'Favori',
+            'Encryption'                          : 'Chiffrement',
+            'On the server'                       : 'Sur le serveur',
+            'Custom Fields'                       : 'Champs personnalisés',
+            'Secret'                              : 'Secret',
+            'File'                                : 'Fichier',
+            'Value'                               : 'Valeur',
+            'Toggle visibility'                   : 'Changer la visibilité',
+            'Generate password'                   : 'Générer un mot de passe',
+            'Simple Server Side Encryption V1'    : 'Version 1 simple chiffrement coté serveur',
+            'Take some notes'                     : 'Prendre quelques notes',
+            'Save'                                : 'Sauvegarder',
+            'Create folder'                       : 'Créer un dossier',
+            'Folder created'                      : 'Dossier crée',
+            'Creating folder failed'              : 'Le dossier n’a pas pu être créé',
+            'Rename folder'                       : 'Renommer le dossier',
+            'Folder renamed'                      : 'Dossier renommé',
+            'Renaming folder failed'              : 'Le dossier n’a pas pu être renommé',
+            'Folder moved'                        : 'Dossier déplacé',
+            'Moving folder failed'                : 'Le dossier n’a pas pu être déplacé',
+            'Delete folder'                       : 'Effacer le dossier',
+            'Do you want to delete the folder'    : 'Êtes vous sur de vouloir effacer le dossier ?',
+            'Folder deleted'                      : 'Dossier effacé',
+            'Deleting folder failed'              : 'Le dossier n’a pas pu être effacé',
+            'Folder restored'                     : 'Dossier restauré',
+            'Restoring folder failed'             : 'Le dossier n’a pas pu être restauré',
+            'Create tag'                          : 'Créer une étiquette',
+            'Tag created'                         : 'Étiquette crée',
+            'Creating tag failed'                 : 'L’étiquette n’a pas pu être effacé',
```

Comment:
One of the (many) tricky rules in French : in this case, `effacé` should take a `e` at the end => `L’étiquette n’a pas pu être effacée`